src/SpotifyApi.NetCore/Models/Playlist.cs

```diff
@@ -269,4 +269,16 @@ public partial class PlaylistPaged
         [JsonProperty("total")]
         public int Total { get; set; }
     }
+
+    /// <summary>
+    /// Upon adding items to a playlist a snapshot_id is returned.
+    /// </summary>
+    public partial class PlaylistSnapshotID
```

Comment:
Please create a separate class file for this and rename to `ModifyPlaylistResponse`. See this example: https://github.com/Ringobot/SpotifyApi.NetCore/pull/40/files#diff-3070e73922144bda6f13360673a6810cR5